spec/system/admin/users_spec.rb

```diff
@@ -0,0 +1,43 @@
+# frozen_string_literal: true
+
+describe 'Users', :js do
+  let(:url) { '/admin/users' }
+  let(:downloaded_csv) { 'users.csv' }
+
+  it_behaves_like 'a page restricted to super admins'
+  it_behaves_like 'a page that can export data'
+  it_behaves_like 'an ActiveScaffold page', %w[Show Edit], 'Users'
```

Comment:
You can add the `search` action:
```ruby
%w[Show Edit Search]
```

Reply:
No I can't. Or rather, I can, but it breaks the spec. The first link that the text 'Search' matches is the link to search.gov, which is not the link we want. The 'acts like a search' behavior that I removed at your request was specifically designed to cope with this,

I don't think this is a good idea anyway. The actions in that array are _per-user_ actions, not global to the page.